### PR TITLE
add fromAmount and showFees url params

### PR DESCRIPTION
**Pull request by Sharqiewicz**
Closes: #321

### CODE REVIEW

**Comment by Sharqiewicz** [comment]
@pendulum-chain/devs ready for review ✅

File: src/pages/swap/useSwapUrlParams.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  setShowFeeCollapse: StateUpdater<boolean>;
+}
+
+export const useSwapUrlParams = ({ form, setShowFeeCollapse }: UseSwapUrlParamsProps) => {
+  useEffect(() => {
+    const params = new URLSearchParams(window.location.search);
+
+    const fromAmountParam = params.get('fromAmount');
+    if (fromAmountParam) {
+      const parsedAmount = Number(fromAmountParam);
```
[the remark this answers is not in the document]
Comment (Sharqiewicz) [reply src/pages/swap/useSwapUrlParams.ts:17]: Well spotted @gianfra-t

**Review by gianfra-t — APPROVED** [review]
Look good. The `showFee` param is not working for me but the logic seems fine.

**Comment by Sharqiewicz** [comment]
> showFee

@gianfra-t it is `showFees` :D 
I applied the changes you mentioned

**Comment by ebma** [comment]
Code looks good but I also can't seem to make the fee comparison open with the parameter. I used
https://deploy-preview-333--pendulum-pay.netlify.app/?fromAmount=12&showFees=true

@Sharqiewicz any ideas?

**Comment by Sharqiewicz** [comment]
@ebma Changed the params. `fromAmount` and `showCompareFees` are now the valid ones.

**Comment by prayagd** [comment]
@Sharqiewicz Tested them and works for True and False and also some amount combinations all work

but tried adding to=usdt and that does not work, is that expected on deploy preview? https://deploy-preview-333--pendulum-pay.netlify.app/?fromAmount=100&showCompareFees=true&to=usdt

**Comment by Sharqiewicz** [comment]
@prayagd `to=usdt` is not included in this PR. The code for `to=usdt` is included in the separate production release

**Comment by prayagd** [comment]
Cool then, i would say good to merge from my end

**Comment by Sharqiewicz** [comment]
@pendulum-chain/devs Can I merge directly to production?

**Comment by gianfra-t** [comment]
> Can I merge directly to production?

Fine by me, if it's needed quickly there.

**Comment by ebma** [comment]
Sure, let's go.

**Pull request merged**